app/main.py

```diff
@@ -1,3 +1,12 @@
 def cache(func):
-    # Write your code here
-    pass
+    lis = []
+
+    def inner(*args):
+        if args not in lis:
+            lis.append(args)
```

Comment:
`If decorated function runs with repeating arguments it should return stored result instead of calling the function again. Decorator cache creating for decorating only functions that take immutable arguments.`